packages/vite/types/importMeta.d.ts

```diff
@@ -50,14 +52,20 @@ interface ImportMeta {
 
   readonly env: ImportMetaEnv
 
-  glob(pattern: string): Record<
+  glob(
+    pattern: string,
+    options?: AssertOptions
```

Comment:
Sorry for the late review. But we can't import types in this ambient type module. It will fail to provide completion for Vite client types. So we need to import like `import('../src/node/importGlob').AssertOptions`.

Reply:
Also can we have ESLint warns about type importing? I couldn't find like `/* eslint-warn @typescript-eslint/consistent-type-imports */`

Reply:
@ydcjeff thanks for being on top of this, would you make a PR with the needed changes?